htmlcss/index_with_styling.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Teaching Materials: HTML &amp; CSS</title>
+  <link href="http://twitter.github.com/bootstrap/assets/css/bootstrap.css" rel="stylesheet">
+  <link href="../common/styles.css" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="stylez.css" typle="text/css">
```

Comment:
Can you add a comment here so that students figure out where the new styles come from?